ceph-dashboard-nightly/build/build

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -e
+n_build_jobs=$(get_nr_build_jobs)
+n_test_jobs=$(($(nproc) / 4))
+export CHECK_MAKEOPTS="-j${n_test_jobs} -N -Q"
```

Comment:
there is no need to actually "make check", as all PR should have run it. you could set `CHECK_MAKEOPTS=--version` to bypass the tests

Reply:
Ack.

Reply:
For master we use ```run-make.sh``` instead of ```run-make-check.sh```.
For nautilus we use ```CHECK_MAKEOPTS='-N -Q'``` instead of ```--version```
as it is more clear that we don't want to run the tests:
```
ctest -h

-N,--show-only              = Disable actual execution of tests.
-Q,--quiet                      = Make ctest quiet.
```